14/umbraco-commerce/release-notes/v14.0.0.md

```diff
@@ -0,0 +1,126 @@
+
+  The only exception to this is the `PaymentController` which is used as a callback for payment gateways. This is currently kept at the same URL to prevent breakages for any existing transaction communications, however all new integrations should communicate with a new Payments API.
+
+## UI Extension Points
+
+With the new backoffice comes a new extensions system for the UI and we have tried to use this to the maximum. These means converting the old UI configs system to using the new manifests system.
+
+With the new extension points it is possible to:
+
+* Change the properties used by the order editor.
+* Add properties to the order line properties editor.
+* Add properties to the Notes, Additional Info and Customer Details modals.
+* Add properties to the order collection view.
+* Add analytics widgets (still in progress)
+* Define custom views for properties to control value rendering.
+
+Here is an example of how you would configure such properties
+
+```javascript
+export const manifests : Array<UcManifestOrderLineProperty> = [
+    {
+        type: 'ucOrderLineProperty',
+        alias: 'Uc.OrderLineProperty.Color',
+        name: 'Order Line Color',
+        weight: 100,
+        meta: {
+            propertyAlias: 'color',
+            editorUiAlias: 'Umb.PropertyEditorUi.EyeDropper',
+            labelElementName: 'uc-mini-color-swatch'
+        }
+    },
+    {
+        type: 'ucOrderLineProperty',
+        alias: 'Uc.OrderLineProperty.GiftMessage',
+        name: 'Order Line Gift Message',
+        weight: 100,
+        meta: {
+            propertyAlias: 'giftMessage',
+            editorUiAlias: 'Umb.PropertyEditorUi.TextArea',
+            summaryStyle: 'table'
+        }
+    }
+];
+```
+
+With these extension points, the Umbraco Commerce order editor is now more flexible than it has ever been before.
+
+{% hint style="info" %}
+We are yet to launch an Umbraco Commerce NPM module so these points are not yet currently available to use externally but we wanted to give you a heads up of what is coming so you can prepare.
+{% endhint %}
+
+## Management API
+
+As with the CMS, with the new UI comes a whole new API layer.
+
+![Umbraco Commerce Management API](../media/v14/management-api.png)
+
+With the Umbraco Commerce API we have aimed to keep it aligned with the Storefront API with support for filtering an expansion where possible.
+
+A key thought whilst developing this API has been to ensure external developers might use this to build other UI's for Umbraco Commerce such as a dedicated mobile app and so we've tried to ensure there are no "special" endpoints just for the Umbraco CMS UI.
+
+## Localization
+
+One of the things that has been on the Umbraco Commerce TODO list for a while is to add backoffice localization support. Given the fresh start it was only logical that we take the time to ensure that everything we built fully supported localization so this will be included in this release.
```

Comment:
```suggestion
Given the fresh start, we made sure that everything we built fully supports localization, so this will be included in this release.
```